src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,121 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationService registrationService = new RegistrationServiceImpl();
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LOGIN_LENGTH = 5;
+    private static final int MIN_PASSWORD_LENGTH = 5;
```

Comment:
The `MIN_PASSWORD_LENGTH` should be 6 according to the checklist requirement for the minimum password length.